index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Green-on-Mind Design Designing Sustainably</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+        href="https://fonts.googleapis.com/css2?family=PT+Sans:wght@400;700&family=Sansita+Swashed:wght@600;800&display=swap"
+        rel="stylesheet">
+</head>
+
+<body>
+    <nav>
+        <ul>
+            <li>
+                <a href="#details">details</a>
+
+            </li>
+            <li>
+                <a href="#speakers">speakers</a>
+            </li>
+            <li>
+                <a href="#register">register</a>
+            </li>
+        </ul>
+    </nav>
+
+    <h1>Green-on-Mind Design</h1>
+
+    <section id="details">
+        <h2>- details-</h2>
+    </section>
+    <p><strong>theme:</strong> Designing Sustainably</p>
+    <p><strong>location:</strong> Virtually</p>
+    <p><strong>when:</strong> April 22, 2021</p>
+    <p><strong>time:</strong> 12PM - 3PM (EST)</p>
+    <p><strong>who's invited:</strong> Designers</p>
+    <p><strong>what:</strong> Green-on-Mind is proud to present the speakers and date of its next monthly event! As
+        always, it will be virtual
+        to lower attendees' carbon footprint to travel, save money and resources, and to broaden accessibility for
+        anyone wanting to join. It is primarily for designers, as speakers will refer to design terminology, but anyone
+        is welcome. In this conference, we will hear from three speakers: Emma Fanning, Michelle Fehler, and Suhaila
+        Sikand, focusing on ways designers can ensure their practice is more sustainable.</p>
```

Comment:
If this heading is in reference to the content that follows, the `section` element can and should wrap around all of this:
```suggestion
        <p><strong>theme:</strong> Designing Sustainably</p>
        <p><strong>location:</strong> Virtually</p>
        <p><strong>when:</strong> April 22, 2021</p>
        <p><strong>time:</strong> 12PM - 3PM (EST)</p>
        <p><strong>who's invited:</strong> Designers</p>
        <p><strong>what:</strong> Green-on-Mind is proud to present the speakers and date of its next monthly event! As
        always, it will be virtual
        to lower attendees' carbon footprint to travel, save money and resources, and to broaden accessibility for
        anyone wanting to join. It is primarily for designers, as speakers will refer to design terminology, but anyone
        is welcome. In this conference, we will hear from three speakers: Emma Fanning, Michelle Fehler, and Suhaila
        Sikand, focusing on ways designers can ensure their practice is more sustainable.</p>
    </section>
```

Reply:
The two comments above about the dashes and `section` elements apply to the code below, too.